poetry/console/commands/init.py

```diff
@@ -373,7 +373,7 @@ def _parse_requirements(
         for requirement in requirements:
             requirement = requirement.strip()
             extras = []
-            extras_m = re.search(r"\[([\w\d,-_]+)\]$", requirement)
+            extras_m = re.search(r"\[([\w\d,-_ ]+)\]$", requirement)
```

Comment:
Shoud we just use `\s` here?

Reply:
The range of `\s` is to wide. I don't think supporting e.g. tab characters here.

Reply:
Hmm, I guess for this case it is fine (ish). For PEP 508, whitespace typically refers to "non-line breaking whitespace". That is narrowier than `\s` and but wider than ` `. I doubt anyone is going to type in `foo[a, \t b]` :smirk: